src/displayapp/screens/Bird.cpp

```diff
@@ -0,0 +1,143 @@
+#include "displayapp/screens/Bird.h"
+#include "displayapp/DisplayApp.h"
+#include "displayapp/LittleVgl.h"
+#include "displayapp/screens/Symbols.h"
+
+#include <cstdlib> // for rand()
+
+using namespace Pinetime::Applications::Screens;
+
+Bird::Bird() {
+
+  lv_obj_t* background = lv_obj_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(background, LV_HOR_RES, LV_VER_RES);
+  lv_obj_set_style_local_radius(background, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, 0);
+  lv_obj_set_style_local_bg_color(background, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x2874a6));
+
+  cactus_top = lv_obj_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(cactus_top, CACTUS_WIDTH, CACTUS_HEIGHT);
+  lv_obj_set_style_local_bg_color(cactus_top, LV_OBJ_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GREEN);
+  lv_obj_set_style_local_border_color(cactus_top, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_obj_set_style_local_border_width(cactus_top, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, 1);
+
+  cactus_bottom = lv_obj_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(cactus_bottom, CACTUS_WIDTH, CACTUS_HEIGHT);
+  lv_obj_set_style_local_bg_color(cactus_bottom, LV_OBJ_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GREEN);
+  lv_obj_set_style_local_border_color(cactus_bottom, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_obj_set_style_local_border_width(cactus_bottom, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, 1);
+
+  MovePipe();
+
+  points = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(points, "    ");
+  lv_obj_align(points, lv_scr_act(), LV_ALIGN_IN_TOP_RIGHT, -10, 5);
+
+  info = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(info, "touch to start");
+  lv_obj_align(info, lv_scr_act(), LV_ALIGN_IN_TOP_MID, 0, 40);
+
+  bird = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(bird, Symbols::dove);
+  lv_obj_set_style_local_text_color(bird, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_YELLOW);
+  lv_obj_set_pos(bird, BIRD_X, pos);
+
+  taskRefresh = lv_task_create(RefreshTaskCallback, LV_DISP_DEF_REFR_PERIOD, LV_TASK_PRIO_MID, this);
+}
+
+Bird::~Bird() {
+  lv_task_del(taskRefresh);
+  lv_obj_clean(lv_scr_act());
+}
+
+void Bird::MovePipe() {
+  lv_obj_set_pos(cactus_top, cactus_x, -cactus_y_offset);
+  lv_obj_set_pos(cactus_bottom, cactus_x, CACTUS_HEIGHT + CACTUS_GAP - cactus_y_offset);
+  return;
+}
+
+void Bird::Refresh() {
+  if (is_stopped) {
+    return;
+  }
+
+  pos += accel / 6;
+
+  if (is_ascending) {
+    if (accel >= -18) {
+      accel -= 3;
+    }
+  } else {
+    if (accel <= 42) {
+      accel++;
```

Comment:
Same as your other PR. I think adding variable names to the magic numbers would be a good idea.